src/implementations/vanilla/components/form/form.scss

```diff
@@ -22,37 +22,22 @@
   margin: 0;
   padding: 0;
 
-  > *:not(.ecl-form-label--hidden) + * {
+  > *:not(.ecl-u-sr-only) + * {
     margin-top: map.get(theme.$spacing, 'xs');
   }
 
-  > *:not(.ecl-form-label--hidden) + .ecl-checkbox {
+  > *:not(.ecl-u-sr-only) + .ecl-checkbox,
```

Comment:
I think i don't understand the use case we are dealing with here...where do we have an `ecl-u-sr-only` element in the definition of our forms? I'd say that if we don't use utilities in components then we shouldn't even use them in their style, but i'm sure i'm missing something here... :)

Reply:
I tracked the original rules setting to https://github.com/ec-europa/europa-component-library/pull/1035
It seems to me there are specific rules to follow which we better keep in order to avoid breaking some spacing in edge cases.

Reply:
Regarding the usage of utility classes within components' styling: seems that the linter is ok and there are no "weak" rules although the mixed specificity. I don't know why but i think it's fine for this case